panel/src/components/Navigation/PageTree.vue

```diff
@@ -77,11 +77,12 @@ export default {
 		},
 		async preselect(page) {
 			// get array of parent uuids/ids
-			const parents = await this.$panel.get("site/tree/parents", {
+			const response = await this.$panel.get("site/tree/parents", {
 				query: {
 					page
 				}
 			});
+			const parents = response.data;
```

Comment:
Maybe more simple writing 🤔 

```suggestion
			const parents = await this.$panel.get("site/tree/parents", {
				query: {
					page
				}
			}).data;
```

Reply:
I think I like your other version better (the one you used in your commit). The `.data` at the end of the expression might be easier overlooked.